From 6ff322280bafa826bd07260cdeda926586b31f0f Mon Sep 17 00:00:00 2001
From: Alexey Nikandrov <anikandrov@habana.ai>
Date: Sun, 06 Apr 2025 10:23:16 +0300
Subject: [PATCH] [SW-224734] package: qual - add missing lib

Added a new dependency entry for libtpc_numerics in the Gaudi3 filelist.
This change ensures that the Gaudi3 package can correctly load and use the libtpc_numerics library.

Change-Id: I2f8225a99f406066f9eb63e5a0f584e1006a235d
Signed-off-by: Alexey Nikandrov <anikandrov@habana.ai>
(cherry picked from commit 7f81a439c72769869682b9122d4e5bcd6ba25b49)
---

diff --git a/packages-specs/qual/gaudi3-filelist.yaml b/packages-specs/qual/gaudi3-filelist.yaml
index 4413e95..a4209ca 100644
--- a/packages-specs/qual/gaudi3-filelist.yaml
+++ b/packages-specs/qual/gaudi3-filelist.yaml
@@ -47,3 +47,5 @@
     dst: /opt/habanalabs/qual/gaudi3/lib/libcoral_user_gaudi3.so
   - src: ${FULLDEPS_DIR}/coral-release/${BUILD_OS}/libpsoc_g3.so
     dst: /opt/habanalabs/qual/gaudi3/lib/libpsoc_g3.so
+  - src: ${FULLDEPS_DIR}/sw-release/${BUILD_OS}/libtpc_numerics.so
+    dst: /opt/habanalabs/qual/gaudi3/lib/libtpc_numerics.so
\ No newline at end of file
